### ISSUE TITLE
Utility of lane_number: time to deprecate?

### ISSUE THREAD

**Issue by j-d-b** [usdot-jpo-ode/wzdx#153]
The optional `lane_number` property on the [Lane](https://github.com/usdot-jpo-ode/jpo-wzdx/blob/master/spec-content/objects/Lane.md) object was left in the major v3.0 release as there were reservations about giving a lane with a not typically driveable lane type (e.g. `shoulder` or `sidewalk`) a number. For clarity and to remove connection to what individuals think of when they see "lane number", the required lane `order` property was added. `order` is used as an index to identify the lanes positions (order) relative to each other. For many contributors/producers, this was the utility of `lane_number`, so with `order`, `lane_number` has no value. I support that conclusion and think that `lane_number` remaining in the spec just adds confusion. 

In summary, it is not clear that there is any utility in `lane_number`.

If there are any producers or consumers who see value in `lane_number` aside from identifying the lane's position, please raise your opinions!

If there is no value, the next step is to deprecate `lane_number`.

**Comment by j-d-b** [comment]
Every time I present the spec, I always gloss over `lane_number` as for new adopters it adds confusion rather than value. 

Seeing again no new comments here, I'll move forward with the PR.

**Comment by natedeshmukhtowery** [comment]
Thumbs up.

**Comment by j-d-b** [comment]
Completed in #175.
